tools/docker/cpu_tests.Dockerfile

```diff
@@ -4,8 +4,8 @@ COPY tools/install_deps/tensorflow-cpu.txt ./
 RUN pip install -r tensorflow-cpu.txt
 
 RUN apt-get update && apt-get install -y sudo rsync
-COPY tools/install_deps/bazel.sh ./
-RUN bash bazel.sh
+COPY tools/install_deps/bazel_linux.sh ./
+RUN bash bazel_linux.sh
```

Comment:
Will this also work for macOS?

Reply:
Yes, that's the magic of Docker :D

Reply:
Off course it will! Forgot it was docker for a moment. Was misled by the presence of `bazel_macos.sh` in the same folder.